docs/source/gcp/examples/appengine-certificates.rst

```diff
@@ -0,0 +1,28 @@
+App Engine - Check if an SSL Certificate is About to Expire
+============================================================
```

Comment:
app engine certificates are automatically rotated and platform managed, why is this useful.

Reply:
@kapilt, the ones used by default indeed are, however, it is also possible to use custom ones:
https://cloud.google.com/appengine/docs/standard/python/securing-custom-domains-with-ssl#using_your_own_ssl_certificates